ParkingZoneApp/Areas/Admin/Controllers/ParkingZonesController.cs

```diff
@@ -92,18 +92,18 @@ public IActionResult Edit(Guid id, ParkingZone parkingZone)
 
                 return RedirectToAction(nameof(Index));
             }
-
-            return View(parkingZone);
+        
```

Comment:
space?

Reply:
I was tought to put a space after braces